docs/agent-reference/agent-core/configuration-manager.md

```diff
@@ -0,0 +1,95 @@
+# Device Update Agent Configuration
```

Comment:
Note: I don't see a discussion of what startup looks like

Reply:
We will use the existing start flow. The difference is forking child agent process according to agents collection in config file.